src/main/java/duke/commands/HelpCommand.java

```diff
@@ -0,0 +1,32 @@
+package src.main.java.duke.commands;
+
+/**
+ * Help function that helps the user to clarify doubts
+ */
+public class HelpCommand extends Command {
+
+    public static final String COMMAND_WORD = "help";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
+            + "Example: " + COMMAND_WORD;
+
+    @Override
+    public CommandResult execute() {
+        String[] stringArray = {src.main.java.duke.commands.AddTodoCommand.MESSAGE_USAGE,
+            src.main.java.duke.commands.AddEventCommand.MESSAGE_USAGE,
+            src.main.java.duke.commands.AddDeadlineCommand.MESSAGE_USAGE,
+            src.main.java.duke.commands.DeleteCommand.MESSAGE_USAGE,
+            src.main.java.duke.commands.MarkDoneCommand.MESSAGE_USAGE,
+            HelpCommand.MESSAGE_USAGE,
+            src.main.java.duke.commands.ExitCommand.MESSAGE_USAGE};
```

Comment:
This is just personal preference but it may be better to import all the relevant Command classes at the top to make things a little easier to read. 
Eg: Instead of 'src.main.java.duke.commands.AddTodoCommand.MESSAGE_USAGE', import the AddTodoCommand class above and store 'AddTodoCommand.MESSAGE_USAGE'.

Reply:
Will correct it. Thank you.